pkg/controller/ingress/ingress_controller.go

```diff
@@ -328,6 +339,74 @@ func (r *ReconcileIngress) Reconcile(request reconcile.Request) (reconcile.Resul
 
 }
 
+func (r *ReconcileIngress) verifyAndUpdateASG(instance *extensionsv1beta1.Ingress, remove bool) error {
```

Comment:
High Cohesion and low coupling : this method has two distinct functions attaching a targetGroup to ASG and detaching them from the ASG, it would improve readability if we split this method along the same lines, attachTGtoASG() and detachTGfromASG(). IF the TG exists we can print the same and return

Reply:
Fixing